### ISSUE TITLE
Add new column with parameters for optimization result file.

### ISSUE THREAD

**Issue by kenorb** [EA31337/FX-MT-VM#48]
The goal is to parse generated report html file and modify before conversion to the text file, so it consist extra column with the current optimized settings.

Currently when running backtest via `run_backtest.sh` script (`-t` & `-o`), it is generates the following text format from html file:

              Pass  Profit Total trades Profit factor Expected Payoff Drawdown $ Drawdown %
                 3 9448.94        60771          1.23            0.16    5093.25      35.09
                 2 9448.94        60771          1.23            0.16    5093.25      35.09
                 1 9447.97        60771          1.23            0.16    5093.25      35.10

See: [_optimization_results](https://github.com/EA31337/EA31337-Lite-Sets/tree/master/EURUSD/spread-10/digits-5/2014/_optimization_results).

We need to parse html format and copy settings value into separate column at the end.

So basically we need to add a new column into generated optimization result html report with settings used for each row, 

<img width="473" alt="screen shot 2016-03-29 at 00 49 50" src="https://cloud.githubusercontent.com/assets/266306/14093781/59de6b8c-f548-11e5-8e41-b9ffec0ec2aa.png">

The idea is to use the following `ex` command to parse the data:

    :g/td title/ exec 'norm f"l"q3yW/<.td><.tr>' . "\<CR>f>a<td>\<Esc>" . 'qp' . "a</td>\<Esc>"

but it needs some work.

Est. 4-6h

--

[Archive.zip](https://github.com/EA31337/FX-MT-VM/files/417859/Archive.zip)
[Archive2.zip](https://github.com/EA31337/FX-MT-VM/files/417860/Archive2.zip)

**Comment by kenorb** [comment]
Fix:

    sed 's/ title="\(.*;\).*"\(.*\)<\/tr>/\2<td>\1<\/td><\/tr>/g'

**Comment by kenorb** [comment]
Fixed in c43a8232f65dcbcd22523b164386ee1b703457b0
